changelog/1.23.0.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "1.23.0"
+description: "Released on 09/22/2021"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.23.0.
+
+### Features ✨
+
+- Doctor: Doctor assesses a Kubernetes cluster to determine its readiness for
+  Coder installation; if there are issues, Doctor provides information on why.
+- Terraform: Coder has released the first of a series of
+  [Terraform scripts](https://github.com/cdr/enterprise-terraform) to facilitate
+  one-click Coder deployment.
+
+- web: added support for generic applications; users can provide images with
+  custom applications, instead of only using applications that have been built
+  into the container image `PATH`.
```

Comment:
Would we be able to link to the docs here?

Reply:
We can as soon as they're ready